src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,42 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import React from 'react';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  return (
+    <ul>
+      {tabs.map(tab => (
+        <li
+          className={classNames(
+            { 'is-active': tab.id === selectedTabId },
+          )}
+          data-cy="Tab"
+          key={tab.id}
+        >
+          <a
+            href={`#${tab.id}`}
+            data-cy="TabLink"
+            onClick={() => {
+              if (tab.id !== selectedTabId) {
+                onTabSelected(tab);
+              }
+            }}
+          >
+            {tab.title}
+          </a>
+        </li>
+      ))}
+    </ul>
+  );
+};
```

Comment:
```suggestion
}) => (
    <ul>
      {tabs.map(tab => (
        <li
          className={classNames(
            { 'is-active': tab.id === selectedTabId },
          )}
          data-cy="Tab"
          key={tab.id}
        >
          <a
            href={`#${tab.id}`}
            data-cy="TabLink"
            onClick={() => {
              if (tab.id !== selectedTabId) {
                onTabSelected(tab);
              }
            }}
          >
            {tab.title}
          </a>
        </li>
      ))}
    </ul>
  );
```
you may use shorthand return